packages/manifest-utils/test/index.ts

```diff
@@ -1,2 +1,3 @@
 /// <reference path="../../../typings/index.d.ts" />
 import './getSpecFromPackageManifest.test'
+import './updateProjectManifestObject.test'
```

Comment:
this whole file is not needed. It was needed when we used tape.

Reply:
done~